### PR TITLE
Scrcmd_pokemon_misc_data

### PR DESCRIPTION
**Pull request by PikalaxALT**
Must come after #273

### CODE REVIEW

**Review by tgsm — DISMISSED** [review]

**Comment by red031000** [comment]
will review this one after the one before is sorted

File: src/field/scrcmd_pokemon_misc.c [hunk 1]
```diff
@@ -89,13 +91,24 @@ static void ov01_0220116C(SCR_648_STRUCT *unkPtr);
 static void GetHiddenPowerPowerType(Pokemon *mon, s32 *power, s32 *type);
 static LocalMapObject *ov01_02201F98(MapObjectManager *mapObjectManager, u8 unkA, u16 species, u16 form, u32 gender, u32 x, u32 y, u32 mapId);
 
-extern u16 ov01_02209AE0[10];
+// zukan_data_XXXXX
```
Comment (adrienntindall) [inline src/field/scrcmd_pokemon_misc.c:94]: zukan = pokedex so this list is...?

File: src/field/scrcmd_pokemon_misc.c [hunk 2]
```diff
@@ -41,6 +41,8 @@
 #include "unk_02037C94.h"
 #include "sound_02004A44.h"
 #include "constants/map_sections.h"
+#include "msgdata/msg.naix"
+// #include "application/zukanlist/zukan_data/zukan_data.naix"
```
Comment (adrienntindall) [inline src/field/scrcmd_pokemon_misc.c:45]: Any reason why this is commented out and not just removed?

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: src/field/scrcmd_pokemon_misc.c (hunk 2 above)
Comment (PikalaxALT) [reply src/field/scrcmd_pokemon_misc.c:45]: Because the naix does not exist yet.

File: src/field/scrcmd_pokemon_misc.c (hunk 1 above)
Comment (PikalaxALT) [reply src/field/scrcmd_pokemon_misc.c:94]: References into zukan_data.narc

**Review by red031000 — APPROVED** [review]
wondering if some of the data could use consts, but otherwise it's good, understandable that you'd want to dump the narc seperately

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**